_signal_processing/krakenSDR_signal_processor.py

```diff
@@ -445,59 +441,61 @@ def run(self):
                                     freq_list.append(write_freq)
                                     doa_result_log_list.append(doa_result_log)
                                 else:
```

Comment:
The data is saved only when DoA is not invoked (disabled, below squelch threshold or have no actives VFOs). Is it intentional?

Reply:
Yes, it is collection of all DOA/IQ sample during long transaction